README.md

```diff
@@ -175,7 +175,8 @@ var storage = multer.diskStorage({
     cb(null, '/tmp/my-uploads')
   },
   filename: function (req, file, cb) {
-    cb(null, file.fieldname + '-' + Date.now() + '-' + Math.random())
+    const uniqueHash = Date.now() + '-' + Math.random().split('.')[1]
```

Comment:
```suggestion
    const uniqueHash = Date.now() + '-' + Math.random().toString().split('.')[1]
```

Reply:
@jonchurch I already did that, you somehow reviewed an old copy?

Reply:
Yep looks like it, ignore this hehe